backend/tracim_backend/lib/core/content.py

```diff
@@ -49,6 +49,7 @@
 from tracim_backend.exceptions import WorkspacesDoNotMatch
 from tracim_backend.lib.core.notifications import NotifierFactory
 from tracim_backend.lib.core.userworkspace import RoleApi
+from tracim_backend.lib.search.models import SimpleContentSearchResponse
```

Comment:
I suggest to rename `SimpleContentSearchResponse` to `SimpleContentSearchResult`?

Reply:
By the way, the search feature must be moved to `lib.search.xxx` module. There is no reason for `lib.core.content` having a dependency on `lib.search.models`.

What you should do is:

- have a *simple search* module which has a dependency on `lib.core.content` and which implements the search method.
- have a dependency on `lib.search.models` in *simple search* module
- have a `lib.core.content` module which does not known anything about search.

Reply:
Is the new implementation ok ?